osu.Game/OsuGame.cs

```diff
@@ -575,7 +575,7 @@ protected override void LoadComplete()
             Container logoContainer;
             BackButton.Receptor receptor;
 
-            dependencies.CacheAs(idleTracker = new GameIdleTracker(6000));
+            dependencies.CacheAs(idleTracker = new GameIdleTracker(300_000, Dependencies.Get<SessionStatics>()));
```

Comment:
Shouldn't be changing this dependency injected instance, since it is used elsewhere in the game and relies on the 6seconds setting.

Instead, you can make a second instance of `GameIdleTracker` below this, specifically for the session statics handling.